### ISSUE TITLE
npm install not installing all dependencies

### ISSUE THREAD

**Issue by batjko** [philcockfield/ui-harness-sample#3]
Hi there.

After doing an `npm install`, which appears to have worked fine (no errors), `npm start` gives me the following error, apparently unable to find `babel-polyfill`:

``` powershell
> npm start                                                                                                                  

> my-components@0.0.1 start c:\Git\uiharness-sample                                                                          
> node ./node_modules/ui-harness/start                                                                                       

module.js:339                                                                                                                
    throw err;                                                                                                               
    ^                                                                                                                        

Error: Cannot find module 'babel-polyfill'                                                                                   
    at Function.Module._resolveFilename (module.js:337:15)                                                                   
    at Function.Module._load (module.js:287:25)                                                                              
    at Module.require (module.js:366:17)                                                                                     
    at require (module.js:385:17)                                                                                            
    at Object.<anonymous> (c:\Git\uiharness-sample\node_modules\ui-harness\lib\server\index.js:7:1)                          
    at Module._compile (module.js:435:26)                                                                                    
    at Object.Module._extensions..js (module.js:442:10)                                                                      
    at Module.load (module.js:356:32)                                                                                        
    at Function.Module._load (module.js:311:12)                                                                              
    at Module.require (module.js:366:17)                                                                                     

npm ERR! Windows_NT 6.1.7601                                                                                                 
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"  
npm ERR! node v4.2.2                                                                                                         
npm ERR! npm  v2.14.7                                                                                                        
npm ERR! code ELIFECYCLE                                                                                                     
npm ERR! my-components@0.0.1 start: `node ./node_modules/ui-harness/start`                                                   
npm ERR! Exit status 1                                                                                                       
npm ERR!                                                                                                                     
npm ERR! Failed at the my-components@0.0.1 start script 'node ./node_modules/ui-harness/start'.                              
npm ERR! This is most likely a problem with the my-components package,                                                       
npm ERR! not with npm itself.                                                                                                
npm ERR! Tell the author that this fails on your system:                                                                     
npm ERR!     node ./node_modules/ui-harness/start                                                                            
npm ERR! You can get their info via:                                                                                         
npm ERR!     npm owner ls my-components                                                                                      
npm ERR! There is likely additional logging output above.                                                                    

npm ERR! Please include the following file with any support request:                                                         
npm ERR!     c:\Git\uiharness-sample\npm-debug.log                                                                                                                                                                                                        
```

I have tried manually installing babel-polyfill, but then it complains about a different package being missing and I don't really want to do this one by one, when the existing package.json should include all of these.

System Info:
Windows 7 (64bit)
node: 4.2.2
npm: 2.14.7

**Comment by philcockfield** [comment]
Hi there @batjko

UIHarness runs on Node 5+ stable (at the moment).  I wonder if it's that.  It should throw a helpful warning message to that effect, but I wonder if it's hitting this `babel-polyfill` issue prior to being able to do the check (if indeed, that's the problem).

**Comment by batjko** [comment]
Hi @philcockfield 
Yes, upgrading to the latest Node (5.7, incl. npm 3.6) did the trick:
![image](https://cloud.githubusercontent.com/assets/7287076/13430006/f9cd4850-dfba-11e5-9d94-61422a7c2259.png)

It's complaining about an internal path problem, which is unrelated though. Everything seems to work fine now.

Thanks for the tip!

**Comment by philcockfield** [comment]
Sweet @batjko - that's `WARNING` is by design.  Although it doesn't look like a weird path.  Probably the back-slash (`\`) on windows as opposed to `/` on unix.

It's also just a warning - shouldn't be causing problems.

**Comment by xurban42** [comment]
Hi there! Same issue on node 3.8.0 (Mac OS X, npm 3.7.3)

```
$ npm start

> my-components@0.0.1 start /ui-harness-sample
> node ./node_modules/ui-harness/start

module.js:341
    throw err;
    ^

Error: Cannot find module 'babel-polyfill'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/ui-harness-sample/node_modules/ui-harness/lib/server/index.js:7:1)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v5.8.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE
npm ERR! my-components@0.0.1 start: `node ./node_modules/ui-harness/start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the my-components@0.0.1 start script 'node ./node_modules/ui-harness/start'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the my-components package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./node_modules/ui-harness/start
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs my-components
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls my-components
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!    /ui-harness-sample/npm-debug.log
$ npm -v
3.7.3
$ node -v
v5.8.0
```

**Comment by philcockfield** [comment]
Ouch! I reproduced that last night over here. Will push an update shortly.
Thanks Martin.

**Comment by philcockfield** [comment]
@xurban42 just published `ui-harness@3.7.6`.  This should be working now.  Please let me know if it's not for you.

Thanks for the heads up @xurban42

**Comment by philcockfield** [comment]
Hold on - still seeing an issue with this.  Fixing now.
